### ISSUE TITLE
WMS URL in the map legend is the one of the internal service and not the one exposed on the net

### ISSUE THREAD

**Issue by elpaso** [g3w-suite/g3w-admin#135]
Using version 3.x in the docker compose LTR environment from https://github.com/g3w-suite/g3w-suite-docker.

**How to reproduce:**
- open a map
- right+click on a vector layer in the legend
+ copy WMS url

![Screenshot_20200910_153749](https://user-images.githubusercontent.com/142164/92738645-6a1cfb80-f37c-11ea-8f33-df1af7a9c74c.png)

**Actual (wrong):**
http://localhost:8080/api/config/1/qdjango/1

**Expected**: 
http://my.external.url/api/config/1/qdjango/1

**Comment by wlorenzetti** [comment]
@elpaso _expected_ should be:
http://my.external.url/ows/1/qdjango/1/

/api/config is API for client config.

**Comment by elpaso** [comment]
You are right, my attention was captured by the first part of the error: the wrong domain and port.

**Comment by Raruto** [comment]
@wlorenzetti is this still a bug?

Testing on https://v35.g3wsuite.it/it/map/demo-35/, it seems to me that the copied url is like what you say (ie. `/ows/`)

**Comment by Raruto** [comment]
Closing this due to inactivity.
